extensions/emmet/src/abbreviationActions.ts

```diff
@@ -201,7 +217,8 @@ function doWrapping(individualLines: boolean, args: any) {
 		}
 		return '';
 	}
-	const abbreviationPromise = (args && args['abbreviation']) ? Promise.resolve(args['abbreviation']) : vscode.window.showInputBox({ prompt: 'Enter Abbreviation', validateInput: inputChanged });
+	let value = individualLines ? previousAbbreviationIndividualLines : previousAbbreviation;
```

Comment:
You can just assume `individualLines = true` now, so there's no need for this check